bound/kt/src/main/kotlin/web5/sdk/dids/methods/jwk/DidJwk.kt

```diff
@@ -1,47 +1,46 @@
 package web5.sdk.dids.methods.jwk
 
 import web5.sdk.crypto.keys.Jwk
+import web5.sdk.crypto.keys.KeyManager
+import web5.sdk.crypto.keys.ToInnerKeyManager
+import web5.sdk.crypto.keys.ToOuterKeyManager
+import web5.sdk.dids.BearerDid
 import web5.sdk.dids.Did
 import web5.sdk.dids.Document
 import web5.sdk.dids.ResolutionResult
+import web5.sdk.rust.Dsa
+import web5.sdk.rust.didJwkCreate
+import web5.sdk.rust.didJwkCreate as rustCoreJwkCreate
+import web5.sdk.rust.DidJwkCreateOptions as RustCoreDidJwkCreateOptions
 import web5.sdk.rust.didJwkResolve as rustCoreDidJwkResolve
-import web5.sdk.rust.DidJwk as RustCoreDidJwk
+
+data class DidJwkCreateOptions(
+    val keyManager: KeyManager? = null,
+    val dsa: Dsa? = null
```

Comment:
Can Jwk have X25519 public keys, which are not digital signature algs?

Reply:
Not to bikeshed, but DSA may be the wrong abstraction, or maybe just the wrong name

Reply:
I had the same thought, let's think about it. I'm not sure there is a good answer as the taxonomy lines are blurry. Maybe we should just do a `CryptoPrimitive` to match the spec? https://tbd54566975.github.io/web5-spec/#cryptographic-primitives